framework_tvb/tvb/interfaces/rest/server/facades/operation_facade.py

```diff
@@ -107,10 +107,8 @@ def launch_operation(self, current_user_id, model_file, project_gid, algorithm_m
                 for key, value in adapter_instance.get_form_class().get_upload_information().items():
                     data_file = fetch_file(request_file_key=key, file_extension=value)
                     data_file_path = save_temporary_file(data_file, temp_folder)
-                    file_name = os.path.basename(data_file_path)
                     upload_field = getattr(view_model_h5, key)
-                    upload_field.store(os.path.join(storage_path, file_name))
-                    shutil.move(data_file_path, storage_path)
+                    upload_field.store(data_file_path)
```

Comment:
Won't be enough to do this change because uploaders run async and the temp folder is cleaned up in the `finally` block.